clang/test/AST/HLSL/ast-dump-comment-cbuffer-tbuffer.hlsl

```diff
@@ -43,8 +43,8 @@ tbuffer B {
 // AST-NEXT: FullComment
 // AST-NEXT: ParagraphComment
 // AST-NEXT: TextComment {{.*}} Text=" CBuffer decl."
-// AST-NEXT: VarDecl {{.*}} a 'float'
-// AST-NEXT: VarDecl {{.*}} b 'int'
+// AST-NEXT: VarDecl {{.*}} a 'hlsl_constant float'
+// AST-NEXT: VarDecl {{.*}} b 'hlsl_constant int'
```

Comment:
Might we want a different address space for tbuffer, potentially?

Reply:
Correct. I will update this test to be just for `cbuffer` and will handle `tbuffer` separately.